deploy/servctl_utils/deploy_command_utils.py

```diff
@@ -40,6 +42,7 @@
 
 SECRETS = {
     'elasticsearch': ['users', 'users_roles', 'roles.yml'],
+    'es-snapshot-gcs': ['{deploy_to}/gcs.client.default.credentials_file'],
```

Comment:
shouldn't our credentials for accessing google cloud be the same in both environments?

Reply:
Also, given that this secret is referenced in our elasticsearch yml file and is used by ECK, is there a reason you chose to make this its own secret and not part of the main elasticsearch secret?

Reply:
No, the service account we're creating only grants access to a particular GCS bucket -- we don't want the dev ES cluster to have storage.admin on the prod cluster's snapshot bucket.

Reply:
> Also, given that this secret is referenced in our elasticsearch yml file and is used by ECK, is there a reason you chose to make this its own secret and not part of the main elasticsearch secret?

I wanted to avoid rocking the elasticsearch boat, since our deployment methodology deletes all the secrets and re-creates them. But this was before I realized that it was going to restart the cluster to add the `secretSetting`... so I'm happy to move this into the elasticsearch-secrets namespace, given that the boat is already rocking 😄

Reply:
Yeah I totally get the no-boat-rocking thing, but sadly that does seem necessary here so 🤞

Reply:
Just following up on this, the snapshot secrets *must* remain separate. Because of the way ECK auto-injects the `secureSettings` into the cluster trust store, it tries to also inject all the other items in the elasticsearch-secrets as well -- not all of which are valid entries in the trust store. Result is crashy elasticsearch :/